sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -133,6 +141,41 @@ public CalcitePlanner(FrameworkConfig config)
     this.context = config.getContext();
     this.connectionConfig = connConfig(context);
     this.typeSystem = config.getTypeSystem();
+
+    // With the catalog, schemas can provide functions.
+    // Add the necessary indirection. The type factory used here
+    // is the Druid one, since the per-query one is not yet available
+    // here. Nor are built-in function associated with per-query types.
+    this.operatorTable = new ChainedSqlOperatorTable(
```

Comment:
I wonder if this is new functionality - could it be in a separate PR?

Reply:
Thanks! removed this